colortv/docs/androidTVSDK.md

```diff
@@ -792,19 +801,196 @@ Check our example [item layouts](https://github.com/color-tv/android-SampleApp/t
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_tvTitle
+* **View type:** `TextView`
+* **Description:** Displays the video title.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDescription
+* **View type:** `TextView`
+* **Description:** Displays the video description.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDuration
+* **View type:** `TextView`
+* **Description:** Displays the video duration.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvSeries
+* **View type:** `TextView`
+* **Description:** Displays series name when recommending series.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvSeason
+* **View type:** `TextView`
+* **Description:** Appends series season number to contained by the view text, while recommending series. I.e. when TextView has set in XML file a text "Season: " it will append season nb. creating "Season: 1".
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvEpisode
+* **View type:** `TextView`
+* **Description:** Appends series episode number to contained by the view text, while recommending series. I.e. when TextView has set in XML file a text "Episode: " it will append episode nb. creating "Episode: 1".
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_clickable
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `Button`)
+* **Description:** Element used to capture click on recommendation, which invokes `onContentChosen` callback. If this element is absent `ctv_upNext` is such element. It should be used only when having other views in UpNext which can be focusable and clickable. For TVs it should use some selector in order to point out an user that focus has been changed.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_btnClose
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`)
+* **Description:** Element used to close UpNext on click. For TVs it should use some selector in order to point out an user that focus has been changed.
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_openRecCenter
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`)
+* **Description:** Element used to close UpNext and open Recommendation Center on click. For TVs it should use some selector in order to point out an user that focus has been changed.
+* **Device:** ALL
+<br /><br />
+
+#### **ID:** ctv_upNextContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `FrameLayout`)
+* **Description:** Used to automatically locate UpNext fragment at a bottom of a screen when it's parent is `RelativeLayout` or `FrameLayout`. If the view is an instance of `LinearLayout` and contains a views with ids: `ctv_space` and `ctv_upNext`, then it sets weight of that elements to take half of a screen on TVs and tablets, and full screen on mobiles. 
```

Comment:
I it would be for default layout only it wouldn't be here. Its for enablig default locating at a bottom and making its width to half of a screen for TVs and tablets (as written in doc)